client/post-editor/editor-ground-control/index.jsx

```diff
@@ -351,33 +339,34 @@ export default React.createClass( {
 							{ this.translate( 'Save' ) }
 						</button>
 					}
-					{ this.props.isSaving &&
-						<span className="editor-ground-control__saving">
-							{ this.translate( 'Saving…' ) }
+					{ ! this.isSaveEnabled() &&
+						<span className="editor-ground-control__save-status">
+							{ this.getSaveStatusLabel() }
 						</span>
 					}
 				</div>
-				{
-					this.state.showAdvanceStatus &&
-						<EditPostStatus
-							savedPost={ this.props.savedPost }
-							postDate={ postDate }
-							type={ this.props.type }
-							onSave={ this.props.onSave }
-							onTrashingPost={ this.props.onTrashingPost }
-							onDateChange={ this.setPostDate }
-							site={ this.props.site }>
-						</EditPostStatus>
-				}
 				<div className="editor-ground-control__action-buttons">
-					<button
-						className="editor-ground-control__preview-button button"
+					<Button
+						borderless
+						className="editor-ground-control__preview-button"
 						disabled={ ! this.isPreviewEnabled() }
 						onClick={ this.onPreviewButtonClick }
 						tabIndex={ 4 }
 					>
-						{ this.getPreviewLabel() }
-					</button>
+						<Gridicon icon="visible" /> <span className="editor-ground-control__button-label">{ this.getPreviewLabel() }</span>
+					</Button>
+					<Button
+						borderless
+						className="editor-ground-control__toggle-sidebar"
+						onClick={ this.props.toggleSidebar }
+					>
+						<Gridicon icon="cog" /> <span className="editor-ground-control__button-label">
+							{ this.translate( '{{type/}} Settings', {
```

Comment:
This issue has been brought up in prior comments above [[#](https://github.com/Automattic/wp-calypso/pull/11536#issuecomment-283617792)], [#](https://github.com/Automattic/wp-calypso/pull/11536#issuecomment-283616405)].  It would be ideal if we could lean on post-types labels for a proper translation to use here.  Here is a list of `labels` for Testimonials for example:

<img width="425" alt="new_testimonial_ _testingtimmy2_wordpress_com_ _wordpress_com" src="https://cloud.githubusercontent.com/assets/22080/23730862/d96ff7d6-041e-11e7-9c8f-e59de42ef636.png">

I don't really see a label that works well in this context.  Perhaps could we use a general generic word that would work in all post types?  Throughout this long PR thread, the sidebar has been referred to as "Options" in many comments.  Could we simply have the word "Options" next to the cog?  @mtias, @jasmussen, @folletto thoughts?